project/legislation/baseclass.py

```diff
@@ -0,0 +1,85 @@
+"""
+The base class from which all ProPublica and OpenSecrets functions derive their power,
+from the base client to interacting with theirrespective APIs and/or django databases.
+"""
+
+import .json
+import .logging
+import .httplib2
+#this goes with the congresscritters class
+from django.db import models
+
+class PClient(object):
+    """
+    PClient deals with fetching responses from eith the ProPublica API, or the OpenSecrets API
+    , and dealing with what is returned.
+
+    The httplib2 stuff is shamelessly stolen from StackOverflow
```

Comment:
When using code from StackOverflow, which is quite alright to do, just include a link to the relevant answer. I.e. give attribution. Consider also upvoting both the question and answer.